nest-asyncio 1.5.6

skupr-anaconda opened the pull request:
**Jira ticket:** [PKG-943](https://anaconda.atlassian.net/browse/PKG-943)

**The upstream data:**
Changes: https://github.com/erdewit/nest_asyncio/compare/v1.5.5...v1.5.6
License: https://github.com/erdewit/nest_asyncio/blob/v1.5.6/LICENSE
Requirements:
- https://github.com/erdewit/nest_asyncio/blob/v1.5.6/pyproject.toml
- setup.cfg https://github.com/erdewit/nest_asyncio/blob/v1.5.6/setup.cfg
- 


**_Actions:_**

- Add --no-deps flag to script

- Remove pinnings from host

- Add pip check


**Additional info:**

<details>

**Package's statistics**

<details>

 * Priority C | effort: easy | Category: other | subcategory:  | pkg_type: python
 * Outdated platfroms: 'linux-64', 'linux-aarch64', 'linux-ppc64le', 'osx-64', 'osx-arm64', 'win-64'
 * Latest version: 1.5.6
 * Release on PyPi:
    * version: 1.5.6
    * date: 2022-09-30T07:09:16
* [PyPi history](https://pypi.org/project/nest-asyncio/#history)
 * Popularity: 
    * 3 months downloads: 1921677.0
    * All time:  7449551 downloads -  nest-asyncio  1.5.6  Patch asyncio to allow nested event loops  

</details>

**Other checks:**

<details>

1. - [x] https://github.com/erdewit/nest_asyncio/tree/v1.5.6 exists
3. - [x] Check the pinnings    
8. - [x] Verify that the `build_number` is correct
9. - [x] has `setuptools`
10. - [x] has `wheel`
11. - [x] `pip` in test
    
12. - [x] Verify the test section
13. - [x] Verify if the package is `architecture specific`
14. - [x] Verify that private modules are not mentioned in the recipe For example: (_private_module)
15.  - [x] license_file: LICENSE is present

16. - [x] license_family BSD is present
17. - [x] License: BSD-2-Clause
    - [x] License is `spdx` compliant
 * Check if the license identifier has correct name from the SPDX License List

| Identifier          |
|:--------------------|
| BSD-2-Clause        |
| BSD-2-Clause-Patent |
| BSD-2-Clause-Views  |
</details>

**Check dependency issues:**

<details>
../aggregate/nest-asyncio-feedstock/recipe/meta.yaml
Dependencies: ['setuptools >=42', 'wheel', 'toml', 'setuptools_scm >=3.4.3', 'pip', 'python']


noarch:
- py3.7:  No issues found
- py3.8:  No issues found
- py3.9:  No issues found
- py3.10:  No issues found

linux-64:
- py3.7:  No issues found
- py3.8:  No issues found
- py3.9:  No issues found
- py3.10:  No issues found

linux-ppc64le:
- py3.7:  No issues found
- py3.8:  No issues found
- py3.9:  No issues found
- py3.10:  No issues found

linux-s390x:
- py3.7:  No issues found
- py3.8:  No issues found
- py3.9:  No issues found
- py3.10:  No issues found

osx-64:
- py3.7:  No issues found
- py3.8:  No issues found
- py3.9:  No issues found
- py3.10:  No issues found

osx-arm64:
- py3.8:  No issues found
- py3.9:  No issues found
- py3.10:  No issues found

win-64:
- py3.7:  No issues found
- py3.8:  No issues found
- py3.9:  No issues found
- py3.10:  No issues found

</details>

**Package Build Score: 19**


**Links:**
* [Anaconda Recipes feedstock](https://github.com/AnacondaRecipes/nest-asyncio-feedstock)
* [Update branch](https://github.com/AnacondaRecipes/nest-asyncio-feedstock/tree/1.5.6)
* [conda-forge recipe](https://github.com/{upstream}/{feedstock_name}-feedstock)
* [PyPI](https://pypi.org/project/nest-asyncio)
* [Diff between upstream and feature branch](https://github.com/conda-forge/nest-asyncio-feedstock/compare/main...AnacondaRecipes:1.5.6)
* [Diff between origin and feature branch](https://github.com/AnacondaRecipes/nest-asyncio-feedstock/compare/master...AnacondaRecipes:1.5.6)
* [The last merged PRs](https://github.com/AnacondaRecipes/nest-asyncio-feedstock/pulls?q=is%3Apr+is%3Amerged+sort%3Aupdated-desc+)

</details>


**Updating the recipe:**
If the recipe needs additional modification the update branch can be modified. Note that the PR diffs are not updated with these changes.
```
git clone -b 1.5.6 git@github.com:AnacondaRecipes/nest-asyncio-feedstock.git
```

skupr-anaconda committed:
DOR update to 1.5.6

anaconda-pkg-build commented:
<details><summary>Linter check found the following problems:</summary>
The following problems have been found:

ERROR: /tmp/abs_a3kevrupnb/clone/recipe/meta.yaml:1: missing_pip_check: For pypi packages, pip check should be present in the test commands
WARNING: /tmp/abs_a3kevrupnb/clone/recipe/meta.yaml:46: missing_license_url: The recipe is missing a license_url
WARNING: /tmp/abs_a3kevrupnb/clone/recipe/meta.yaml:46: missing_doc_source_url: The recipe is missing a doc_source_url
Errors were found

</details>

skupr-anaconda committed:
Release 1.5.6

- Add --no-deps flag to script

- Remove pinnings from host

- Add pip check

[PKG-943]

anaconda-pkg-build commented:
<details><summary>Linter check found the following problems:</summary>
The following problems have been found:

WARNING: /tmp/abs_6cpvusbozc/clone/recipe/meta.yaml:50: missing_doc_source_url: The recipe is missing a doc_source_url
WARNING: /tmp/abs_6cpvusbozc/clone/recipe/meta.yaml:50: missing_license_url: The recipe is missing a license_url
Warnings were found

</details>

psteyer reviewed (approved):
LGTM

varlackc reviewed (commented):
This might be up to debate but it might be a good idea in this case to point the `doc_url` to the `README` file. 

for example

https://github.com/erdewit/nest_asyncio/blob/master/README.rst

skupr-anaconda committed:
Update doc_url

skupr-anaconda commented:
> https://github.com/erdewit/nest_asyncio/blob/master/README.rst

Thanks. I've updated doc_url

anaconda-pkg-build commented:
<details><summary>Linter check found the following problems:</summary>
The following problems have been found:

WARNING: /tmp/abs_27m7qfekkr/clone/recipe/meta.yaml:50: missing_license_url: The recipe is missing a license_url
WARNING: /tmp/abs_27m7qfekkr/clone/recipe/meta.yaml:50: missing_doc_source_url: The recipe is missing a doc_source_url
Warnings were found

</details>

anaconda-pkg-build commented:
<details><summary>Linter check found the following problems:</summary>
The following problems have been found:

WARNING: /tmp/abs_89ayu88ecc/clone/recipe/meta.yaml:50: missing_doc_source_url: The recipe is missing a doc_source_url
WARNING: /tmp/abs_89ayu88ecc/clone/recipe/meta.yaml:50: missing_license_url: The recipe is missing a license_url
Warnings were found

</details>
